resources/main.css

```diff
@@ -287,15 +287,28 @@ button,
 }
 
 .developer-mode-content button {
+    font-size: 16px;
     background: white;
     flex: auto;
     padding: 4px;
     appearance: button;
     border: 2px solid rgba(255, 255, 255, 0.5);
     border-radius: 10px;
 }
+
+.developer-mode-content button span {
+    padding-left: 10px;
+    font-size: 25px;
+    line-height: 10px;
+    position: relative;
+    top: 2px;
```

Comment:
I understand you're trying to center them but that doesn't work well:
![image](https://github.com/user-attachments/assets/4750d3ee-ff04-4261-ac79-7c643067b270)

The main issue is that the button isn't high enough I believe?

I think you want something like this on the button:
```css
display: inline-flex;
align-items: center;
justify-content: center;
```

But I couldn't make it _very_ centered. I always found it difficult to center font icons rather than real icons TBH...

So I'm fine if it's not perfect either.

Reply:
Here on my mac this looks reasonably centered on all browsers:

Safari: <img width="476" alt="Screenshot 2025-05-27 at 16 21 12" src="https://github.com/user-attachments/assets/b51553a8-1f86-4157-b179-cd88e5d3dcef" />
Chrome:  <img width="471" alt="Screenshot 2025-05-27 at 16 21 29" src="https://github.com/user-attachments/assets/f0e74747-4a7e-4eb5-b527-d9d2be74a794" />
Firefox: <img width="457" alt="Screenshot 2025-05-27 at 16 21 41" src="https://github.com/user-attachments/assets/79c16675-ef3c-4d6b-b5d4-8b040ea458de" />


I guess the rest will be down to custom fonts with different base positions.